### PR TITLE
Add support for persistent-2.14 to serversession-backend-persistent

### LINKED ISSUE #38: serversession-backend-persistent doesnt compile with 2.14.*
The changes in persistent 2.14.0.0 break the instance used by serversession-backend-persistent. 

Working on a PR to allow both 2.13.* and 2.14.* to compile

### PR DESCRIPTION
**Pull request by belevy**
Fixes #38 

Bumped version to 2.0.2

### CODE REVIEW

**Comment by ncaq** [comment]
Is `toPersistValues` on line 121 a typo for `toPersistFields`?

File: serversession-backend-persistent/src/Web/ServerSession/Backend/Persistent/Internal/Impl.hs [hunk 1]
```diff
@@ -105,12 +110,21 @@ instance forall sess. P.PersistFieldSql (Decomposed sess) => P.PersistEntity (Pe
...
 
+#if MIN_VERSION_persistent(2,14,0)
   toPersistFields (PersistentSession a b c d e) =
+    [ P.toPersistValue a
+    , P.toPersistValue b
+    , P.toPersistValue c
+    , P.toPersistValue d
+    , P.toPersistValue e ]
+#else
+  toPersistValues (PersistentSession a b c d e) =
```
Comment (ncaq) [inline serversession-backend-persistent/src/Web/ServerSession/Backend/Persistent/Internal/Impl.hs]: Is `toPersistValues` a typo for `toPersistFields`?

**Review by ncaq — CHANGES_REQUESTED** [review]

File: serversession-backend-persistent/src/Web/ServerSession/Backend/Persistent/Internal/Impl.hs (hunk 1 above)
Comment (belevy) [reply serversession-backend-persistent/src/Web/ServerSession/Backend/Persistent/Internal/Impl.hs]: Indeed I appear to have been a bit aggressive in my renaming

**Review by ncaq — APPROVED** [review]

**Comment by ncaq** [comment]
Thank you.
I uploaded.

**Pull request merged**